torch/csrc/distributed/rpc/rref_impl.cpp

```diff
@@ -190,13 +190,18 @@ RRefForkData UserRRef::fork() const {
 
 const IValue& OwnerRRef::getValue() const {
   std::unique_lock<std::mutex> lock(mutex_);
-  valueCV_.wait(lock, [this] { return value_.has_value(); });
+  valueCV_.wait(
+      lock, [this] { return value_.has_value() || error_.has_value(); });
```

Comment:
nit: can just call `hasValue` here?

Reply:
We need to break out of the loop if either there's a value or error.
There are other ways to do it, but it's a cheap check, and avoids yet-another-bool in the impl